jarviz-lib/src/test/java/com/vrbo/jarviz/service/MavenArtifactDiscoveryServiceTest.java

```diff
@@ -59,4 +60,19 @@ public void testDiscoverArtifact_Fail() throws ArtifactNotFoundException {
         artifactDiscoveryService.discoverArtifact(artifact);
     }
 
+    @Test
```

Comment:
For this test to be correct shouldn't we remove "throws ArtifactNotFoundException" from the signature?